lib/src/test/kotlin/starknet/data/types/TransactionsTest.kt

```diff
@@ -45,9 +44,6 @@ internal class TransactionsTest {
                     BigInteger("2"),
                 ),
             ),
-            entryPointSelector = Felt(
-                BigInteger("617075754465154585683856897856256838130216341506379215893724690153393808813"),
-            ),
```

Comment:
Why was this removed?

Reply:
invoke tx in version 1 no longer takes in an entryPointSelector, and we only allow to create new transactions in v1